kornia/augmentation/augmentations.py

```diff
@@ -24,7 +37,18 @@ def __repr__(self) -> str:
         repr = f"(p={self.p}, return_transform={self.return_transform})"
         return self.__class__.__name__ + repr
 
-    def forward_flip(self, input: UnionType, flip_func: Callable) -> UnionType:  # type: ignore
+    @staticmethod
+    def get_params(batch_size: int, p: float = 0.5):
+        return get_random_p_params(batch_size, p)
```

Comment:
I would declare this function as `_get_random_p_params` since it won't be part of the exposed api